examples/wheeled_balancing_async.py

```diff
@@ -22,8 +22,9 @@ async def balance(env: gym.Env, logger: mpacklog.Logger):
     observation = env.reset()  # connects to the spine
     action = np.zeros(env.action_space.shape)
     for step in range(1_000_000):
-        observation, reward, done, info = await env.async_step(action)
-        if done:
+        observation, reward, terminated, \
```

Comment:
[black](https://github.com/psf/black) may complain about this, but if that's the case I can take care of applying one commit of linting when we merge :ok_hand: